client/src/core/services/parse-map.service.ts

```diff
@@ -0,0 +1,39 @@
+import { Injectable } from '@angular/core';
+
+@Injectable({
+      providedIn:'root',
+})
+export class ParseMapService {
+    terrain: any = {
+        name: 'Terrain 1',
+        height: 30,
+        width: 50,
+        obstacles: [],
+        trees: [],
+    };
+
+        constructor() { }
+    getMap(): void {
+        return this.terrain;
+    }
+
+    parseFile(file: string): void {
+        let fileTab = file.split('\n');
```

Comment:
Codacy found an issue: ['fileTab' is never reassigned. Use 'const' instead.](https://app.codacy.com/gh/Ca-Cybersecurity-Robiot/robiot/pullRequest?prid=5943029)